### PR TITLE
Update stacks/java-websphereliberty/devfile.yaml

### PR DESCRIPTION
**Pull request by akurinnoy**
### What does this PR do?:

- update schema to `2.2.0`;
- use `https` protocol for the `java-websphereliberty` devfile.

### Which issue(s) this PR fixes:
_Link to github issue(s)_

### PR acceptance criteria:

- [ ] Contributing guide
_Have you read the [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) and followed its instructions?_
- [ ] Test automation
_Does this repository's tests pass with your changes?_
- [ ] Documentation
_Does any documentation need to be updated with your changes?_
- [ ] Check Tools Provider
_Have you tested the changes with existing tools, i.e. Odo, Che, Console? (See [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) on how to test changes)_

### How to test changes / Special notes to the reviewer:

https://workspaces.openshift.com/#https://raw.githubusercontent.com/akurinnoy/registry/patch-1/stacks/java-websphereliberty/devfile.yaml

### CODE REVIEW

File: stacks/java-websphereliberty/devfile.yaml [hunk 1]
```diff
@@ -57,7 +57,7 @@ components:
           path: /
           name: http-websphere
```
Comment (thepetk) [inline stacks/java-websphereliberty/devfile.yaml:58]:
```suggestion
          name: https-websphere
```

**Review by thepetk — CHANGES_REQUESTED** [review]
I've added a minor suggestion regarding the name of the endpoint. @ajm01 as you re listed as an owner of this stack, could you please review the PR?

File: stacks/java-websphereliberty/devfile.yaml (hunk 1 above)
Comment (akurinnoy) [reply stacks/java-websphereliberty/devfile.yaml:58]: @thepetk Thanks for the feedback.

**Review by thepetk — APPROVED** [review]
/lgtm Tested locally with odo 3. I wasn't able to run the `run` command but the failure is not related to this update. As mentioned to my previous comment the PR needs to be approved from the stack owner first in order to be merged.

**Comment by thepetk** [comment]
> /lgtm Tested locally with odo 3. I wasn't able to run the `run` command but the failure is not related to this update. As mentioned to my previous comment the PR needs to be approved from the stack owner first in order to be merged.

@akurinnoy I've revisited my initial review. A minor suggestion is to bump up the version of the devfile too

https://github.com/devfile/registry/blob/0ccd117da09d01078dbb82c1f57393c261f69de1/stacks/java-websphereliberty/devfile.yaml#L34

I'd suggest bumping up to `0.10.0`

**Review by thepetk — COMMENTED** [review]
@ajm01 as you are listed in the OWNERs file could you please review the PR?

**Review by ibuziuk — APPROVED** [review]

**Comment by thepetk** [comment]
> @thepetk thanks, fixed.

@akurinnoy I think there's a small error with the DCO

**Review by svor — APPROVED** [review]

**Pull request merged**